content/200-concepts/100-components/01-prisma-schema/06-relations/420-relation-mode.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: 'Relation mode'
+metaTitle: 'Relation mode'
+metaDescription: 'Relation mode'
```

Comment:
TODO

Reply:
Put something in, happy for better suggestions